pandas/tests/resample/test_datetime_index.py

```diff
@@ -1440,6 +1440,24 @@ def test_downsample_across_dst_weekly():
     tm.assert_series_equal(result, expected)
 
 
+def test_downsample_dst_at_midnight():
+    # GH 25758
```

Comment:
Is the DST transition at midnight on the 4th?

Reply:
Correct https://www.currenttimeonline.com/dst/dst.do?tz=America/Havana